database/src/main/java/com/firebase/ui/database/FirebaseIndexListAdapter.java

```diff
@@ -41,7 +42,7 @@
      */
     public FirebaseIndexListAdapter(Activity activity,
                                     Class<T> modelClass,
-                                    int modelLayout,
+                                    @LayoutRes int modelLayout,
```

Comment:
I also just noticed that we never had an `@Res` annotation on our layouts so I added that too.